js/driver/virtio/net.js

```diff
@@ -96,68 +92,55 @@ function initializeNetworkDevice(pciDevice) {
   }
 
   if (!dev.writeGuestFeatures(features, driverFeatures, deviceFeatures)) {
-    debug('[virtio] driver is unable to start');
-    return;
+    return debug('[virtio] driver is unable to start');
   }
 
-  var hwAddr = dev.netReadHWAddress();
-  var status = dev.netReadStatus();
+  const hwAddr = dev.netReadHWAddress();
+  const status = dev.netReadStatus();
 
-  if (!status) {
-    return;
-  }
+  if (!status) return;
 
-  var QUEUE_ID_RECV = 0;
-  var QUEUE_ID_TRANSMIT = 1;
+  const QUEUE_ID_RECV = 0;
+  const QUEUE_ID_TRANSMIT = 1;
 
-  var recvQueue = dev.queueSetup(QUEUE_ID_RECV);
-  var transmitQueue = dev.queueSetup(QUEUE_ID_TRANSMIT);
+  const recvQueue = dev.queueSetup(QUEUE_ID_RECV);
+  const transmitQueue = dev.queueSetup(QUEUE_ID_TRANSMIT);
   transmitQueue.suppressUsedBuffers();
 
-  function fillReceiveQueue() {
+  const fillReceiveQueue = () => {
```

Comment:
I'd prefer using function declarations instead of anonymous functions (where possible)